server/src/main/java/io/druid/offheap/OffheapBufferPool.java

```diff
@@ -31,7 +31,7 @@
 {
   private static final Logger log = new Logger(OffheapBufferPool.class);
 
-  public OffheapBufferPool(final int computationBufferSize)
+  public OffheapBufferPool(final int computationBufferSize, final int cacheSize)
```

Comment:
I mean move `Offheap` --> `OffHeap` for this class to match the index case change